### PR TITLE
Release/4.7

### PR DESCRIPTION
**Pull request by blms**
Prepare release 4.7 for merge to main.

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,6 +1,30 @@
...
+    - As a content editor, I want to add and edit transcriptions on a separate page from the document detail or admin edit form, so that permissions and saving just the transcription can be managed more easily.
+    - As a content editor I want to select images in the related fragment view in order to determine which images belong with the document.
+    - As a content editor I want to use basic formatting in transcription content so that I can enter lines as numbered lists or tag when the language changes within a document.
+    - As a content editor, when I search for tags to add to a document I want the search to ignore case so that I don't create variations of the same tag.
+    - As a content editor, I want to be able to search for tags with or without diacritics and get the same results.
+    - chore: update transcription editor to use new image+transcription panel viewer
+    - chore: implement Annotorious plugin for side-by-side transcription editing
+    - chore: implement a w3c annotation server to store transcription data
+    - chore: refine logic for identifying transcription chunks that indicate new image
```
Comment (rlskoeser) [inline CHANGELOG.rst:23]: transcription editor features have been tested and accepted but they aren't merged into this branch, so we shouldn't include them (we'll have to make sure we find and track them all down when we do finally merge it in!)

**Review by rlskoeser — COMMENTED** [review]
looks good! we just need to omit the transcription editor features from the changelog since they aren't actually included in this branch

File: CHANGELOG.rst (hunk 1 above)
Comment (blms) [reply CHANGELOG.rst:23]: Ah, good call! I was looking at closed issues first before the git log, thanks for catching this.

File: CHANGELOG.rst [hunk 2]
```diff
@@ -1,6 +1,24 @@
...
+
+- public site
+    - As a user viewing document details I want to see which images are not part of the document so that I understand which parts of the fragment are used for the current document.
+    - As a user looking at images for a single document, I want easy access to documents on images from the same fragment that are not part of the current document.
+
+- content/data admin
+    - As a content editor I want to select images in the related fragment view in order to determine which images belong with the document.
+    - As a content editor, when I search for tags to add to a document I want the search to ignore case so that I don't create variations of the same tag.
+    - As a content editor, I want to be able to search for tags with or without diacritics and get the same results.
+    - chore: refine logic for identifying transcription chunks that indicate new image
```
Comment (rlskoeser) [inline CHANGELOG.rst:17]:
```suggestion
    - refined logic for identifying transcription chunks that indicate new image for `sync_transcriptions` script
```

Comment (rlskoeser) [reply CHANGELOG.rst:17]: I don't think I usually list chores as-is in the change log; will have to think about whether I have a convention for how/if I report them!

**Review by rlskoeser — APPROVED** [review]

**Pull request closed without merging**